tracer.go

```diff
@@ -7,8 +7,9 @@ import "time"
 type Tracer interface {
 
 	// Create, start, and return a new Span with the given `operationName` and
-	// incorporate the given StartSpanOptions. (See StartSpanOption for all of
-	// the options on that front)
+	// incorporate the given StartSpanOption `opts`. (Note that `opts` borrows
```

Comment:
Grammatical grey area. It's not just one StartSpanOption, but that's the type's name.